database/migrations/2022_11_13_212420_create_rooms_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('rooms', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->enum('type', ['STANDARD', 'DUPLEX', 'DELUX']);
+            $table->float('price');
+            $table->enum('bed_type', ['SINGLE', 'DOUBLE', 'KING', 'QUEEN']);
+            $table->integer('bed_count');
+            $table->json('furniture');
+            $table->boolean('balcony');
+            $table->json('services');
+            $table->json('complimentaries');
+            $table->unsignedBigInteger('floor_id');
```

Comment:
From now on, let's move the foreign key under `$table->id` which will help us to manually check the DB tables and understand the relationships easily.